src/lib.rs

```diff
@@ -1386,6 +1571,103 @@ mod tests {
         }
     }
 
+    fn expect_opaque_sequence(
+        input: &[u8],
+        kind: OpaqueSequenceKind,
+        expected_payload: &[u8],
+        expected_trailer: &[Sequence],
```

Comment:
This can just be a boolean to simplify this function and remove the necessity for a constant. `st_terminated: bool` or something like that.